### PR TITLE
Minimize choose time options

### PR DESCRIPTION
**Pull request by yotamshalev**
#### Please Review
- [ ]

### CODE REVIEW

File: src/custom-modules.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+declare module '*.png' {
```
Comment (yotamshalev) [inline src/custom-modules.d.ts:1]: why is this needed?

File: src/components/ChooseTime.tsx [hunk 2]
```diff
@@ -84,36 +69,36 @@ function ChooseTime() {
...
   ]);
 
-  function getTimeAvailability(daysArr: Array<DayInfo>, hoursRanges: Array<HoursRangeInfo>): SessionSubscriptionData['timeAvailability'] {
+  function getTimeAvailability(
+    daysArr: Array<DayInfo>, 
+    hoursRanges: Array<HoursRangeInfo>): SessionSubscriptionData['timeAvailability'] {
     return {
       days: daysArr
-        .filter(dayObj => dayObj.isPressed)
+        // .filter(dayObj => dayObj.isPressed)
```
Comment (yotamshalev) [inline src/components/ChooseTime.tsx:80]: why are you still sending the daysArr

**Review by yotamshalev — COMMENTED** [review]
hey @TerriVic , PR looks good, see my comments below, 
also, I'm not seeing the fix to the success message page text and UI.

File: src/components/ChooseTime.tsx (hunk 2 above)
Comment (TerriVic) [reply src/components/ChooseTime.tsx:80]: I thought you said I should send it as if all of the days are available to preserve the compatibility with choosing days...

File: src/custom-modules.d.ts (hunk 1 above)
Comment (TerriVic) [reply src/custom-modules.d.ts:1]: Because without that I can't import a picture into the typescript file like the day/night icons

File: src/components/ChooseTime.tsx (hunk 2 above)
Comment (yotamshalev) [reply src/components/ChooseTime.tsx:80]: ah ok.. we can do that 👍

**Comment by TerriVic** [comment]
> also, I'm not seeing the fix to the success message page text and UI.

I didn't think it was relevant to this branch, I thought this specific change was more urgent. I'll add it to this PR as well then.
